### PR TITLE
Add new pre-commit hook for Flutter 

### PR DESCRIPTION
**Pull request by guid-empty**
Hello.

Is it possible to add new flutter dependency validation pre-commit hook to be shown then here - https://pre-commit.com/hooks.html?

Hook and some documentation is available here https://github.com/guid-empty/flutter-dependency-validation-pre-commit.

Thank you.

### CODE REVIEW

File: all-repos.yaml [hunk 1]
```diff
@@ -154,6 +154,7 @@
 - https://github.com/Cretezy/flutter-format-pre-commit
 - https://github.com/dluksza/flutter-analyze-pre-commit
 - https://github.com/fluttercommunity/import_sorter
+- https://github.com/guid-empty/flutter-dependency-validation-pre-commit
```
Comment (asottile) [inline all-repos.yaml:157]: please append to the bottom of the file

Comment (guid-empty) [reply all-repos.yaml]: hello.

done.

**Review by asottile — APPROVED** [review]

**Comment by guid-empty** [comment]
@asottile thank you a lot !

**Pull request merged**